julia/NIfTIZarr.jl/src/nii2zarr.jl

```diff
@@ -110,12 +130,9 @@ function _make_pyramid3d(data3d::AbstractArray, nb_levels::Integer, label::Bool=
     nbatch = size(data3d)[1:end-3]
     nxyz = size(data3d)[end-2:end] 
     data4d = reshape(data3d, (prod(nbatch), nxyz...))
-    if nb_levels < 0
-        T = typeof(nb_levels)
-        nb_levels = min(T.(floor.(log2.(nxyz) .- 2))...)
-    end
+    nb_levels = max(nb_levels, 1)
```

Comment:
Do we still make sure that we don't have levels that are smaller than a chunk somewhere?